SampleApp/TangramVC.swift

```diff
@@ -8,16 +8,30 @@
 
 import UIKit
 import TangramMap
-class TangramVC:  MapViewController{
+class TangramVC:  MapViewController, MapMarkerSelectDelegate {
```

Comment:
Class naming should be consistent. Don't abbreviate `ViewController` in `TangramVC`. I don't mind the use of `vc` for inline code but it seems inappropriate for actual class naming.